tests/phpthemis/scell_test.php

```diff
@@ -21,6 +21,12 @@
 
 class ScellTest extends TestCase {
 
+    public function testKeyGeneration() {
+        $defaultLength = 32;
+        $key = phpthemis_gen_sym_key();
+        $this->assertEquals(strlen($key), $defaultLength);
```

Comment:
Will be better if for php we will have more checks, like in themis core, where we check that several calls of function return different keys. because php has a more complicated wrapper and own memory management.